app/components/participant-creator/component.js

```diff
@@ -73,6 +73,12 @@ export default Ember.Component.extend({
             });
             if (!useAsPassword) {
                 password = makeId(10);
+                this.get('extra').pushObject({
+                    key: 'password',
+                    value: password
+                });
+                extra['password'] = password;
+                this.set('useAsPassword', 'password');
```

Comment:
So instead of storing the password in one place, we use a field.... to track the name of another configurable field... that stores the password?

Is there perhaps a more direct approach?